pkg/cluster/internal/create/actions/loadbalancer/loadbalancer.go

```diff
@@ -74,11 +75,19 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 		backendServers[n.String()] = fmt.Sprintf("%s:%d", n.String(), common.APIServerInternalPort)
 	}
 
+	// TODO(bentheelder): settle on a better abstraction for this
+	// While we don't want much code aware of the provider name,
+	// we do need it to configure the external loadbalancer
+	dockerProvider := true
+	if _, ok := ctx.Provider.(*podman.Provider); ok {
+		dockerProvider = false
+	}
 	// create loadbalancer config data
 	loadbalancerConfig, err := loadbalancer.Config(&loadbalancer.ConfigData{
 		ControlPlanePort: common.APIServerInternalPort,
 		BackendServers:   backendServers,
 		IPv6:             ctx.Config.Networking.IPFamily == config.IPv6Family,
+		Docker:           dockerProvider,
```

Comment:
this code should not be provider aware, this sort of thing should be encapsulated in the node provider.